README.md

```diff
@@ -1,3 +1,95 @@
+## Install instructions
+* clone the repo
+* run `bundle` to install the gems
+* you must manually create a `.env` file in the project directory to enable SMS messaging via Twilio. Setup an account with the instructions below. Then set the following `ENV` parameters in `.env` :
+```
+TWILIO_ACC_SID = 'YOUR DETAILS HERE'
+TWILIO_AUTH_TOKEN = 'YOUR DETAILS HERE'
+TWILIO_FROM_NUMBER = 'YOUR DETAILS HERE'
+TWILIO_TO_NUMBER = 'YOUR DETAILS HERE'
+```
+
+## Steps to Run
```

Comment:
Very clear steps on how to run the program. I like that you've done it step by step for each user story.